apps/inheritance_app/inheritance_encrypt_data.c

```diff
@@ -334,6 +429,7 @@ STATIC bool inheritance_encryption_handle_inititate_query(
       !get_wallet_name_by_id(query->encrypt.initiate.wallet_id,
                              (uint8_t *)wallet_name,
                              inheritance_send_error)) {
+    SET_ERROR_FLOW_TAG(ENCRYPTION_INIT_FLOW);
```

Comment:
Shouldn't we put these at the start of every function? rather than only on error?

Reply:
we could do that actually, and that would make error handling more neat. will update this